### ISSUE TITLE
Oxygen can't log in

### ISSUE THREAD

**Issue by LFQO** [thebrumby/HotWalletClaimer#98]
![image](https://github.com/thebrumby/HotWalletClaimer/assets/70878450/597001f7-5848-4023-9a79-fcfecb1cd611)

**Comment by LFQO** [comment]
Button to start is called "play now", but it's not checked

**Comment by thebrumby** [comment]
Hi @LFQO,

The button appears to be called "Start App":

<img width="801" alt="image" src="https://github.com/thebrumby/HotWalletClaimer/assets/29182343/7ce4e83d-c4e4-4787-aa78-8930691078ac">

```
03 July - 03:54 | Need to wait until 03 July - 04:58 before the next claim attempt. Approximately 64 minutes.
Using saved user agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36

CHROME DRIVER INITIALISED: Try not to exit the script before it detaches.
Step 101 - Attempting to check for the start button (should not be present)... [//button[contains(., 'START')]]
Step 102 - Attempting to open a link for the app: //div[contains(@class, 'reply-markup-row')]//button[.//span[contains(text(), 'Start App')]]...
Step 103 - Attempting to click the 'Launch' button (probably not present)... [//button[contains(@class, 'popup-button') and contains(., 'Launch')]]
Step 104 - Attempting to switch to the app's iFrame...
Step 108 - The last lucky box claim was attempted on 03 July 2024, 01:36 AM.
Step 109 - The pre-claim wait time is : Filled and random offset is 5 minutes.
Step 110 - Attempting to click the 'Claim' button... [//div[@class='farm_btn']]
Step 111 - Random offset applied to the wait timer of: 2 minutes.
Step 112 - After BALANCE: 46311.965
STATUS: Successful Claim: Next claim 0h. 59m. 39s. / 61 minutes.

CHROME DRIVER DETACHED: It is now safe to exit the script.
03 July - 04:59 | Need to wait until 03 July - 06:00 before the next claim attempt. Approximately 61 minutes.
```

Everything seems to be claiming correctly for me and the balance is increasing hourly.

**Comment by thebrumby** [comment]
Hi @LFQO,

I’ve been sent some screenshots of the button you describe, even though it appears differently for me. 

I’ve updated the XPATH to match occurrences of either button 😎

**Comment by LFQO** [comment]
Hi, @thebrumby,
Oxygen works now, thank you. I suppose that the name of the button may depend on the language used in telegram.
I also found some problem with Seed. It can't claim because of some error (step 108). The same error remains for mystery box, daily claim and news (steps 105, 106)
![image](https://github.com/thebrumby/HotWalletClaimer/assets/70878450/39166467-3ef0-4911-a293-dc0d1f6f55c2)

**Comment by LFQO** [comment]
Wave don't claim either, but there are no errors in logs.
![image](https://github.com/thebrumby/HotWalletClaimer/assets/70878450/04111825-699f-420c-90c9-e8bf4f8bf203)

Hex-Auto woks almost fine except that it doesn't show balance, when using status

**Comment by LFQO** [comment]
Time-Farm also works strange for me. I'm using two accounts, and it works only on the one of them. Here are logs from both of them.
Working one:
![image](https://github.com/thebrumby/HotWalletClaimer/assets/70878450/cd6aa0da-8c86-433f-bb7b-34ba0c86220c)
Not working one:
![image](https://github.com/thebrumby/HotWalletClaimer/assets/70878450/59fc7938-bbf9-4a8d-892d-277cbcc27ef0)
The accounts are registered in different regions, but it doesn't matter, I guess

**Comment by Wellynounet** [comment]
Hello   a fix for hexacore not showing balance on sttus is released :) go catch with ``./pull-games.sh``

**Comment by thebrumby** [comment]
> Hi, @thebrumby, Oxygen works now, thank you. I suppose that the name of the button may depend on the language used in telegram. I also found some problem with Seed.

Hi, generally the bots are in English, or if there are multilingual versions, they tend to have different bot names, so it's not generally an issue.

I hadn't realised that seed was using a standalone version of the click_element() function in it's code, I've updated and pushed the fix.

**Comment by thebrumby** [comment]
> Time-Farm also works strange for me. I'm using two accounts, and it works only on the one of them. Here are logs from both of them.

The second one is not connecting to the Telegram conversation, so can't interact with the bot.

1) Check that there is no mistyped message in the Telegram GUI with the bot that's preventing it interactng normally. 
2) Consider deleting the conversation with the Bot in Telegram, and starting by adding it again from the beginning. 
3) You may enable the debug setting to save screenshots of what the bot sees, this may give you a clue to why it's going wrong.
4) Delete the glitched game session and make a new one:

```
./launch.sh status
```

```
delete Time-Farm:[session name]
```

**Comment by LFQO** [comment]
Hi, hexacore balance is now showing and seed is working (it seems that seed doesn't update its balance, but i'm not really sure about it). Thank you.
I'm trying to figure out the problem with time-farm (it's really strange that it works only on 1/2 accounts). I enabled debugging, where can I find saved screenshots?
In case if you did something with Wave, it's still not claiming

**Comment by thebrumby** [comment]
@LFQO 

What size PC/VPS are you running the scripts on? Everything is working fine for me on all the games you mention. Some games have no time pressure to get the info, and some need to get it in a split second. It might be a resources issue (or might not).

Screenshots are saved in ./HotWalletBot/screenshots and then your Wallet IDs. e.g. HOT:Wallet1

**Comment by LFQO** [comment]
Does Wave work for you? I'm using docker with 1.5Gb and 2 processor wsl settings. Just for testing, I increased it to 4Gb and 4 processors, but Wave didn't work either.
I think that I found the reason for time-farm working on only one of the accounts. I suppose that this bot gets country code and decides which language to in chat. The working account is registered with +43 code and all conversation is in English. The not working account is registered with +7 and conversation is in Russian. This cause the problem with opening the app, as the logic looks for buttons with specific names. Is there a way to force interaction in english? Or maybe to skip checking the button names for this bot and just pick the first one by default as long as there are only two of them?
